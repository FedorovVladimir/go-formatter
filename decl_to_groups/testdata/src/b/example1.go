package main

var v1 = "v1" // want "rm decl" "incorrect single declaration style"

var v2 = "v2" // want "rm decl" "incorrect single declaration style"
