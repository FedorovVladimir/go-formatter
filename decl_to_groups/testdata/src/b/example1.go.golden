package main

import (
	"fmt" // want "incorrect single declaration style"
)

var (
	v1 = "v1" // want "incorrect single declaration style"
)

const (
	c1 = "c1" // want "incorrect single declaration style"
)

const (
	c2 = "c2" // want "incorrect single declaration style"
)

// doc
const (
	c3 = "c3" // want "incorrect single declaration style"
)

func main() {
	fmt.Println()
}
